src/main/java/codeu/model/data/Activity.java

```diff
@@ -23,9 +26,10 @@
 	*	@param id is the ID of who ever made the activity
 	*	@param creation is the creation time of this activity
 	*/
-	public Activity(String type, UUID id, Instant creation) {
+	public Activity(String type, UUID id, UUID owner, Instant creation) {
```

Comment:
Good job figuring out that each Activity object needed its own id for it to be used correctly in datastore!